content/en/docs/tasks/administer-cluster/encrypt-data.md

```diff
@@ -39,6 +39,8 @@ kind: EncryptionConfiguration
 resources:
   - resources:
       - secrets
+      - configmaps
+      - pandas.awesome.bears.com
```

Comment:
```suggestion
      - pandas.awesome.bears.example
```

Please avoid using real domain names (eg `bears.com`); use a domain reserved for documentation.